cypress/e2e/po/components/ember/ember-accordion.po.ts

```diff
@@ -0,0 +1,29 @@
+import EmberComponentPo from '@/cypress/e2e/po/components/ember/ember-component.po';
+import EmberFormMembersPo from '@/cypress/e2e/po/components/ember/ember-form-members.po';
+
+export default class EmberAccordionPo extends EmberComponentPo {
+  private headerSelector: string;
+  private contentSelector: string;
+
+  constructor(private rootTestId: string) {
+    super(`.accordion:has(> [data-testid="${ rootTestId }__header"])`); // This should work, but given current functionality it isn't used
+    this.headerSelector = `[data-testid="${ this.rootTestId }__header"]`;
+    this.contentSelector = `[data-testid="${ this.rootTestId }__content"]`;
+  }
+
+  expand() {
+    this.header().click();
+  }
+
+  formMembers() {
```

Comment:
Not sure about this, wouldn't we end up adding properties here for every use of the accordion we want to test? Feels kinda backwards like the accordion should be a property on the formMembers class